README.md

```diff
@@ -77,6 +77,34 @@ Then you can run the following commands:
 
```

Comment:
At line 76, there is a full list of arguments, could you add the new proxy argument too?